tests/repos/scripts/hidden_folder

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+# Hide the directory
+cd ..
+mv hidden_folder .hidden_folder
```

Comment:
Looks like some changes to the actual tests are missing from the commit?

Reply:
No, this file add a hidden folder in `KLAUS_REPOS_ROOT`. The already existing test `test_wsgi_autoreload` will only pass if it correctly ignores hidden folders, but no new test case is needed for this feature.

Reply:
However, I did not add a test for the invalid repo display part, I should create one.